# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../setserial/lx_types_h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./setserial.c.orig	Sun Nov 14 12:29:08 1999
+++ ./setserial.c	Sun Nov 14 12:30:09 1999
@@ -22,6 +22,7 @@
 #ifdef HAVE_LINUX_HAYESESP_H
 #include <linux/hayesesp.h>
 #endif
+#include <linux/types.h>
 #include <linux/serial.h>
 
 #define VERSION_STR "2.15"
